che-admin-api/che-core-api-plugin/pom.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2012-2015 Codenvy, S.A.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Codenvy, S.A. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-admin-api-parent</artifactId>
+        <groupId>org.eclipse.che.core</groupId>
+        <version>3.13.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>che-core-api-plugin</artifactId>
```

Comment:
I have doubts about project name and location
@gazarenkov  can you comment this. What is prefered name for this project